docs/products/range-tokens.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Range Tokens
+sidebar_label: Range Tokens
+---
+
+The range token enables a DAO or nascent project to use its native token as collateral to borrow funds. At maturity, if the debt is not paid, the range token holder is instead compensated with an equivalent amount of the collateral (the native token) using the settlement price of the native token to determine the number of tokens. 
```

Comment:
Would be cool to add examples of why a DAO wants to borrow funds. 

Reason: DAO's don't want to sell their token but need funds to pay development costs or pay bounties.

Reply:
Good point. Pulled some content from the medium articles.